build-system/tasks/release/index.js

```diff
@@ -303,8 +312,11 @@ async function populateOrgCdn_(flavorType, rtvPrefixes, tempDir, outputDir) {
   if (flavorType == 'base') {
     rtvCopyingPromises.push(
       ...Object.entries(experimentsConfig)
-        // eslint-disable-next-line local/no-deep-destructuring
-        .filter(([, {environment}]) => environment == 'INABOX')
+        .filter(
+          ([, experimentConfig]) =>
+            'environment' in experimentConfig &&
+            experimentConfig.environment == 'INABOX'
```

Comment:
`experimentConfig?.environment == 'INABOX'`

Reply:
```js
experimentConfig?.environment == 'INABOX' → experimentConfig && experimentConfig.environment == 'INABOX'
￼```

Which is not the same here. I wish there was `experimentConfig.?environment` too :/

Reply:
Less pretty: `experimentConfig.environment ?? '' == 'INABOX'`
I don't feel strongly

Reply:
What are you trying to check here?

Reply:
> Less pretty: `(experimentConfig.environment ?? '') == 'INABOX'`

`experimentConfig.environment` itself is the compilation error - because the type is `ExperimentConfigDef | {}` - `.environment` exists on `ExperimentConfigDef` but not on `{}` - so that wouldn't work without the `in` check...